nixos/tests/acme.nix

```diff
@@ -208,9 +202,9 @@ in import ./make-test-python.nix {
           webserver.succeed(
               "/run/current-system/specialisation/dns-01/bin/switch-to-configuration test"
           )
-          webserver.wait_for_unit("acme-finished-example.com.target")
+          webserver.wait_for_unit("acme-finished-example.test.target")
           client.succeed(
-              "curl --cacert /tmp/ca.crt https://c.example.com/ | grep -qF 'hello world'"
+              "curl --cacert /tmp/ca.crt https://c.example.test/ | grep -qF 'hello world'"
```

Comment:
Given we now have 

```

  security.pki.certificateFiles = [ acme-ca ];
}
```

shouldn't we be able to remove these `--cacert /tmp/ca.crt` calls? If not, why not?

Reply:
For the same reason as my previous reply :)